### ISSUE TITLE
`Receiver::pj_url` is missing fragment parameters

### ISSUE THREAD

**Issue by spacebear21** [payjoin/rust-payjoin#431]
According to the docstring:

```rust
    // The contents of the `&pj=` query parameter.
    // This identifies a session at the payjoin directory server.
    pub fn pj_url(&self) -> Url {
```

But the return value only contains the subdirectory (e.g. `https://payjo.in/4A8JZNN5SD9EV`) and is missing the hpke pubkey, ohttp keys, and expiration in fragment parameters.

**Comment by DanGould** [comment]
I believe this problem was eliminated in #466 by replacing `pj_url` with `subdir`. I don't believe there is a reason to get the complete pj_uri outside of the `pj_uri`.

I'm aware we were using this in Bull Bitcoin mobile to display in the bip21 by applying the `pj=` and `pjos=0` explicitly. @spacebear21 do you think we can refactor Bull Bitcoin mobile to use bitcoin_uri, or do we need some way to extract the request URL explicitly?

**Comment by spacebear21** [comment]
Yeah I think this is obsolete now, we ended up using the pjUriBuilder to display the bip21 anyway. `subdir` is a more descriptive name that I wouldn't expect to contain fragment parameters.
